src/otx/algorithms/visual_prompting/adapters/pytorch_lightning/models/visual_prompters/segment_anything.py

```diff
@@ -227,16 +227,18 @@ def forward(
         if self.config.model.return_single_mask:
             masks, scores = self.select_masks(masks, scores, point_coords.shape[1])
 
-        upscaled_masks = self.mask_postprocessing(masks, orig_size[0])
+        return scores, masks
+        # TODO (sungchul): apply inner postprocessing
+        # upscaled_masks = self.mask_postprocessing(masks, orig_size[0])
 
-        if self.config.model.return_extra_metrics:
-            stability_scores = self.calculate_stability_score(
-                upscaled_masks, self.config.model.mask_threshold, self.config.model.stability_score_offset
-            )
-            areas = (upscaled_masks > self.config.model.mask_threshold).sum(-1).sum(-1)
-            return upscaled_masks, scores, stability_scores, areas, masks
+        # if self.config.model.return_extra_metrics:
+        #     stability_scores = self.calculate_stability_score(
+        #         upscaled_masks, self.config.model.mask_threshold, self.config.model.stability_score_offset
+        #     )
+        #     areas = (upscaled_masks > self.config.model.mask_threshold).sum(-1).sum(-1)
+        #     return upscaled_masks, scores, stability_scores, areas, masks
 
-        return upscaled_masks, scores, masks
+        # return upscaled_masks, scores, masks
```

Comment:
Could you explain about this change?

Reply:
When converting pth to IR, inner postprocessing is malfunctioning. It is required to look at, but it seems to take time. So I temporarily blocked related parts.

Reply:
That means OV model's output is quite different with PyTorch model?

Reply:
Only `upscaled_masks`. Because post-processing `masks` outside of the model looks fine, I think there is little difference of `masks` between torch and IR.
I think inner post-processing is something wrong, but I don't know the reason.

Reply:
In conclusion, postprocessing is done outside of model due to malfunctioning inner postprocessing funciton. Right?

Reply:
Right. These temporary blocks are just to avoid inner post-processed outputs.
outer postprocessing is done at https://github.com/openvinotoolkit/training_extensions/blob/a8c665805ea8cf435ca966fbebf9330dbab68794/src/otx/algorithms/visual_prompting/tasks/openvino.py#L135C1-L141